src/utils/getDefaultDockerContext.ts

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as cp from 'child_process';
+import { DockerOptions } from 'dockerode';
+import * as util from 'util';
+
+const exec = util.promisify(cp.exec);
+
+export async function getDefaultDockerContext(): Promise<DockerOptions | undefined> {
+    try {
+        const { stdout, stderr } = await exec('docker context inspect');
+
+        if (stderr) {
+            throw new Error(stderr);
+        }
+
+        if (!stdout) {
+            return undefined;
+        }
+
+        const dockerContext = JSON.parse(stdout);
+        // tslint:disable: no-unsafe-any

```

Comment:
Define an interface that matches the shape of the returned command, to avoid the linter error as well as help document our expectations?